bikeshed/__init__.py

```diff
@@ -2551,8 +2558,15 @@ def inlineRemoteIssues(doc):
                 if el.tag == "p":
                     el.tag = "div"
                 addClass(el, "no-marker")
-        except:
-            pass
+        except urllib2.HTTPError as err:
+            if doc.token and err.code == 401:
+                die("Unauthorized Access to GitHub's API. There might be an issue with your token.")
```

Comment:
Duh. `Die` sounded so imperious I sort of wrongly assumed it terminated the process. I feel like auth/rate limit should both stop the iteration and return from `inlineRemoteIssues`. I imagine the correct way to do that in python is just to return? Right?